core-bundle/src/Resources/contao/widgets/SerpPreview.php

```diff
@@ -42,8 +44,18 @@ public function generate()
 		$description = StringUtil::substr($this->getDescription($model), 160);
 		$alias = $this->getAlias($model);
 
-		// Get the URL with a %s placeholder for the alias or ID
-		$url = $this->getUrl($model);
+		try
+		{
+			// Get the URL with a %s placeholder for the alias or ID
+			$url = $this->getUrl($model);
+		}
+		catch (ExceptionInterface $routingException)
+		{
+			if ($model->requireItem)
```

Comment:
what if `requireItem` is not set? Do we just show an empty URL?

Reply:
It would actually not throw an exception afaik.

Reply:
Then we do not need the if condition, do we?

Reply:
Yep, removed in e60ed86.